spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -17,6 +17,33 @@ def add_valkyrie_migrations
     rake 'valkyrie_engine:install:migrations'
   end
 
+  def create_collection_resource
+    generate 'hyrax:collection_resource CollectionResource with_basic_metadata'
+
+    gsub_file "config/metadata/collection_resource.yaml", "attributes: {}",
+              <<-YAML
+
+---
+attributes:
+  target_audience:
+    type: string
+    form:
+      primary: true
+      multiple: true
+  department:
+    type: string
+    form:
+      primary: true
+  course:
+    type: string
+    form:
+      primary: false
```

Comment:
Should this also be the config used in dassie?

Reply:
I agree, they should match.

Reply:
Updated so both dassie and the test app have these custom attributes.